components/Navigation/index.js

```diff
@@ -1,16 +1,14 @@
 import { theme } from "@/styles";
 import styled from "styled-components";
 
-export default function Navigation() {
+export default function Navigation({ onCreateClick }) {
   return (
     <StyledNavigation>
       <StyledNavButtonBox>
         <StyledSideButton onClick={() => (window.location.href = "/joined")}>
```

Comment:
this should not be a button, but a Link, since you navigate to another page.

It would be a button if some logic would have to be applied before navigating.